features/attached_enable.feature

```diff
@@ -644,6 +644,15 @@ Feature: Enable command behaviour when attached to an UA subscription
         """
         esm-infra     yes                enabled            UA Infra: Extended Security Maintenance \(ESM\)
         """
+        When I run `ua detach` `with sudo` and stdin `y`
+        Then stdout matches regexp:
+        """
+        Updating package lists
```

Comment:
@lucasmoura,  I get why we emit this, but can we create a JIRA ticket to sort a single apt update call at the end of detach?

Reply:
Yes, that's a good point @blackboxsw
Will do